prow/scripts/cluster-integration/control-plane-gke-integration.sh

```diff
@@ -259,6 +259,15 @@ data:
   auditlog-config-path: "/path"
   auditlog-security-path: "/path"
   auditlog-tenant: "tnt"
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kcp-provisioner-database-encryption
+  namespace: kcp-system
+type: Opaque
+data:
+  secretKey: MTIzNDU2Nzg5MDEyMzQ1Njc4OTAxMjM0NTY3ODkwMTIK
```

Comment:
```suggestion
  secretKey: MTIzNDU2Nzg5MDEyMzQ1Njc4OTAxMjM0NTY3ODkwMTIK #gitleaks:allow
```

Try adding this comment. This should exclude this key from list of detected leaks.